install/Dockerfile.opendistroknn

```diff
@@ -0,0 +1,55 @@
+# Warning! Do not use this config in production! This is only for testing and security has been turned off.
+
+FROM ann-benchmarks
+
+WORKDIR /home/app
+
+# Install Open Distro following instructions from https://opendistro.github.io/for-elasticsearch-docs/docs/install/deb/
+RUN apt-get install software-properties-common -y
+RUN add-apt-repository ppa:openjdk-r/ppa \ 
+    && apt update \
+    && apt install openjdk-11-jdk -y
+RUN apt install unzip -y \
+    && apt-get install wget -y
+RUN wget -qO - https://d3g5vo6xdbdb9a.cloudfront.net/GPG-KEY-opendistroforelasticsearch | apt-key add -
+RUN echo "deb https://d3g5vo6xdbdb9a.cloudfront.net/apt stable main" | tee -a   /etc/apt/sources.list.d/opendistroforelasticsearch.list
+RUN wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-7.9.1-amd64.deb \
```

Comment:
Do you still need elasticsearch-oss if you are also installing opendistro on the next line?

Reply:
Yes, there's a dependency on elasticsearch-oss

`opendistroforelasticsearch : Depends: elasticsearch-oss (= 7.9.1)`